lewis_emulators/flipprps/interfaces/stream_interface.py

```diff
@@ -0,0 +1,20 @@
+from lewis.adapters.stream import StreamInterface, Cmd
+
+
+class FlipprpsStreamInterface(StreamInterface):
+
+    # Commands that we expect via serial during normal operation
+    commands = {
+        Cmd("set_polarity_down", "^dn$"),
+        Cmd("set_polarity_up", "^up$"),
+    }
+
+    terminator = "\r\n"
```

Comment:
I think you might need to change this definition for lewis to recognise it correctly
```suggestion
    in_terminator = "\r\n"
    out_terminator = in_terminator
```